mlflow/environment_variables.py

```diff
@@ -579,13 +579,6 @@ def get(self):
 # How many traces to be buffered at the in-memory trace client.
 MLFLOW_TRACE_BUFFER_MAX_SIZE = _EnvironmentVariable("MLFLOW_TRACE_BUFFER_MAX_SIZE", int, 1000)
 
-#: Whether or not to enable trace logging in model serving.
-#: The default value is set to False to ensure that this flag is only enabled
-#: when our internal safety mechanism on Databricks explicitly sets it to True.
-MLFLOW_ENABLE_TRACE_IN_SERVING = _BooleanEnvironmentVariable(
-    "MLFLOW_ENABLE_TRACE_IN_SERVING", False
-)
-
```

Comment:
We can use `_MLFLOW_ENABLE_TRACE_IN_SERVING` if we just want to hide this from the documentation.

Reply:
Yeah but I removed instead this time as we'll eliminate the env var from MLflow code base eventually (it will only be used inside serving code)🙂

Reply:
sounds good :)